spec/terrapin/runners_spec.rb

```diff
@@ -2,39 +2,39 @@
 
 describe "When picking a Runner" do
   it "uses the BackticksRunner by default" do
-    Terrapin::CommandLine::ProcessRunner.stubs(:supported?).returns(false)
-    Terrapin::CommandLine::PosixRunner.stubs(:supported?).returns(false)
+    expect(Terrapin::CommandLine::ProcessRunner).to receive(:supported?).and_return(false)
+    expect(Terrapin::CommandLine::PosixRunner).to receive(:supported?).and_return(false)
 
     cmd = Terrapin::CommandLine.new("echo", "hello")
 
     expect(cmd.runner.class).to eq(Terrapin::CommandLine::BackticksRunner)
   end
 
   it "uses the ProcessRunner on 1.9 and it's available" do
-    Terrapin::CommandLine::ProcessRunner.stubs(:supported?).returns(true)
-    Terrapin::CommandLine::PosixRunner.stubs(:supported?).returns(false)
+    expect(Terrapin::CommandLine::ProcessRunner).to receive(:supported?).and_return(true)
+    expect(Terrapin::CommandLine::PosixRunner).to receive(:supported?).and_return(false)
 
     cmd = Terrapin::CommandLine.new("echo", "hello")
     expect(cmd.runner.class).to eq(Terrapin::CommandLine::ProcessRunner)
   end
 
   it "uses the PosixRunner if the PosixRunner is available" do
-    Terrapin::CommandLine::PosixRunner.stubs(:supported?).returns(true)
+    expect(Terrapin::CommandLine::PosixRunner).to receive(:supported?).and_return(true)
 
     cmd = Terrapin::CommandLine.new("echo", "hello")
     expect(cmd.runner.class).to eq(Terrapin::CommandLine::PosixRunner)
   end
 
-  it "uses the BackticksRunner if the PosixRunner is available, but we told it to use Backticks all the time" do
-    Terrapin::CommandLine::PosixRunner.stubs(:supported?).returns(true)
+  it "uses the BackticksRunner and don't try PosixRunner, if we told it to use Backticks all the time" do
+    expect(Terrapin::CommandLine::PosixRunner).not_to receive(:supported?)
```

Comment:
I think the old way of stubbing :supported? and returning true was condusing because it never got called.